src/Ops.Web/Areas/Admin/Views/Features/FeatureDetails.cshtml

```diff
@@ -0,0 +1,150 @@
+                                                <a class="nav-link active" data-toggle="tab" href="#md-edit" role="tab">Edit</a>
+                                            </li>
+                                            <li class="nav-item">
+                                                <a class="nav-link" data-toggle="tab" href="#md-preview" role="tab">Preview</a>
+                                            </li>
+                                        </ul>
+                                        <div class="tab-content" id="md-edit-preview">
+                                            <div class="tab-pane show active" id="md-edit" role="tabpanel">
+                                                <div id="wmd-button-bar" class="wmd-button-row"></div>
+                                                <textarea asp-for="Feature.BodyText" value="@Model.Feature.BodyText" class="form-control wmd-input" id="wmd-input"></textarea>
+                                            </div>
+                                            <div class="tab-pane" id="md-preview" role="tabpanel">
+                                                <div id="preview" class="wmd-preview"></div>
+                                            </div>
+                                            <span asp-validation-for="Feature.BodyText" class="text-danger"></span>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <a class="btn btn-secondary" id="return-home" asp-action="@nameof(FeaturesController.Index)">Return to Features</a>
+                    @if (Model.Feature.IsNewFeature)
+                    {
+                        <text>
+                            <button type="submit" id="submit" value="Submit" class="btn btn-primary btn-spinner">
+                                <span class="buttonText">Add Feature</span>
+                                <span class="fa fa-spinner fa-pulse fa-lg fa-fw d-none"></span>
+                            </button>
+                        </text>
+                    }
+                    else
+                    {
+                        <text>
+                            <button type="submit" id="submit" value="Submit" class="btn btn-primary btn-spinner">
+                                <span class="buttonText">Update Feature</span>
+                                <span class="fa fa-spinner fa-pulse fa-lg fa-fw d-none"></span>
+                            </button>
+                        </text>
+                    }
+                </div>
+            </div>
+        </div>
+    </form>
+</div>
+@section scripts{
+    <script src="~/devscripts/fontawesome-iconpicker/main.js" asp-append-version="true"></script>
```

Comment:
References to `devscripts` should only be used wrapped in an if to tell when we're in development mode. Those will not be published to production.